pombola/kenya/static/js/survey.js

```diff
@@ -0,0 +1,75 @@
+(function() {
+
+    // Percentage of users to display the survey for
+    var p = 0.6;
+
+    // Is cookie existing?
+
+    var ct = null;
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
```

Comment:
Could you explicitly hoist the variables in this function? I'm with Douglas Crockford on this - putting a var in the initializer of a for loop when it's really at function scope is confusing. (Similarly, the `var ct` below is unnecessary.)